src/pages/Overview/Filters.ts

```diff
@@ -52,7 +52,7 @@ export const labelFilter: FilterTypeWithFilter<NamespaceInfo> = {
   id: 'label',
   title: 'Label',
   placeholder: 'Filter by Label',
-  filterType: FilterTypes.custom,
+  filterType: FilterTypes.label,
```

Comment:
As discussed  on IRC I think it would be better to have a different filter type here, something like "nsLabel", to not mix up with labels on workloads

Reply:
(since it seems the problem was already there before, I can open a separate ticket)